doc/user/install-operator-sdk.md

```diff
@@ -86,8 +86,8 @@ $ chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin && sudo mkdir -p
 - [go][go_tool] version v1.13+.
 
 ```sh
-$ go get -d github.com/operator-framework/operator-sdk # This will download the git repository and not install it
-$ cd $GOPATH/src/github.com/operator-framework/operator-sdk
+$ git clone https://github.com/operator-framework/operator-sdk
```

Comment:
Users do not need to do the git clone. This would be the recommendation for who would like to contribute with and not just use.